clang/test/SemaCXX/attr-gsl-owner-pointer.cpp

```diff
@@ -131,7 +131,7 @@ template <
     class CharT,
     class Traits>
 class basic_regex;
-static_assert(__is_gsl_pointer(basic_regex<char, void>), "");
+static_assert(__is_gsl_owner(basic_regex<char, void>), "");
```

Comment:
Don't we have a static_assert overload that does not take a second argument?

Reply:
Since C++17, yes. Is it safe to change the test to run in C++17 mode? Most tests seem to just use the default language mode. I copied that part also from https://reviews.llvm.org/D50119

Reply:
I see, makes sense.